src/components/dnn-checkbox/dnn-checkbox.tsx

```diff
@@ -9,7 +15,7 @@ export class DnnCheckbox {
   @Element() el: HTMLDnnCheckboxElement;
 
   /** Defines if the checkbox is checked (true) or unchecked (false) or in an intermediate state (undefined) */
-  @Prop({mutable: true}) checked: boolean | undefined = false;
+  @Prop({mutable: true}) checked: "checked" | "unchecked" | "intermediate" = "unchecked";
```

Comment:
@valadas just a note that the story for `dnn-checkbox` will need to be updated to reflect this change.

Reply:
dnn-checkbox was already updated for this in this same PR. I will though adjust the dnn-checkbox story after rebasing this and in the same PR I guess as this one has the 'breaking' label and explanation in its comment.